submissions/nrushchak/popup/index.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./index.css">
+  <title>Popup</title>
+</head>
+
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <nav class="header__nav">
+        <ul class="header__nav__list">
+          <li class="header__nav__list__item">
+            <a class="header__nav__list__item__link title" href="#">Mail</a>
+          </li>
+          <li class="header__nav__list__item">
+            <a class="header__nav__list__item__link title" href="#">Images</a>
+          </li>
+          <li class="header__nav__list__item">
+            <input class="nav__checkbox__input" type="checkbox" id="popup-checkbox" name="popup-checkbox" />
+            <label for="popup-checkbox" class="nav__checkbox__label">
+              <img class="nav__checkbox__image" src="popup-icons/popup-button.png" alt="popup button" />
+            </label>
+
+            <div class="popup">
```

Comment:
Fix the position of the popup according to the button. Also, fix `z-index` of popup and footer
![image](https://user-images.githubusercontent.com/16196199/187038575-3986e242-2755-4598-a6a0-920123bde8d3.png)